App/Screens/Home/Footer/Footer.tsx

```diff
@@ -63,7 +80,7 @@ export function Footer (props: FooterProps) {
       Share.share({
         title: i18n.t('home_share_title'),
         message: i18n.t('home_share_message', {
-          cigarettes: api!.shootISmoke.cigarettes.toFixed(2)
+          cigarettes: (api && api.shootISmoke.cigarettes.toFixed(2)) || 0
```

Comment:
```suggestion
          cigarettes: api ? api.shootISmoke.cigarettes.toFixed(2) : 0
```

I prefer the `?:` syntax, it reads better imo. Could you replace all the other instances too?